client/src/components/settings/PeriodCustomizationInput.tsx

```diff
@@ -17,7 +17,7 @@ import {periodNameDefault} from '@watt/shared/util/schedule';
 
 type PeriodCustomizationInputProps = {id: string, data: SgyPeriodData};
 export default function PeriodCustomizationInput(props: PeriodCustomizationInputProps) {
-    const {id, data: {n, c, l, o, s}} = props;
+    const {id, data: {n, c, r, o, s}} = props;
```

Comment:
I would generally avoid removing properties on `userData` unless there's 100% chance that they will no longer be needed, because we don't want to clear people's links if we decide they might be useful again.